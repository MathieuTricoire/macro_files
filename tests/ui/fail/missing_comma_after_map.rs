use macro_files::create;

fn main() {
    create!({
        "docs": {
            "README.md": "# Title"
        }
        "LICENSE": "MIT"
    });
}
