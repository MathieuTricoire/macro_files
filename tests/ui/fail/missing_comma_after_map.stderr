error: no rules expected the token `"LICENSE"`
 --> tests/ui/fail/missing_comma_after_map.rs:8:9
  |
7 |         }
  |          - help: missing comma here
8 |         "LICENSE": "MIT"
  |         ^^^^^^^^^ no rules expected this token in macro call
